config/locales/server.en.yml

```diff
@@ -2430,6 +2430,7 @@ en:
 
     shared_drafts_category: "Enable the Shared Drafts feature by designating a category for topic drafts. Topics in this category will be suppressed from topic lists for staff users."
     shared_drafts_min_trust_level: "Allow users to see and edit Shared Drafts."
```

Comment:
Do we want to delete this as well?

Reply:
No, we will delete that at the end with the setting. We have an internal checklist for these at /t/112536